src/components/transactions/transactionsComponent.js

```diff
@@ -9,33 +9,18 @@ class Transactions extends React.Component {
   constructor() {
     super();
     this.state = {
-      transactions: [],
-      offset: 0,
-      loadMore: true,
-      length: 1,
+      canLoadMore: true,
     };
   }
 
-  componentDidMount() {
-    this.loadMore();
-  }
-
   loadMore() {
-    if (this.state.loadMore && this.state.length > this.state.offset) {
-      this.setState({ loadMore: false });
-      transactions(this.props.activePeer, this.props.address, 20, this.state.offset)
+    if (this.state.canLoadMore) {
+      this.setState({ canLoadMore: false });
```

Comment:
As mentioned in official documents:

> After the initial mounting of components, a re-rendering will occur when:
> A component's setState() method is called
> A component's forceUpdate() method is called

So it's a good practice not to use `setState` if you don't intend to re-render the component. In this case, I'll suggest using `this. canLoadMore` instead of `this.state. canLoadMore`.